tbcrawler/pytbcrawler.py

```diff
@@ -55,6 +57,11 @@ def run():
     job_config = ut.get_dict_subconfig(config, args.config, "job")
     job = crawler_mod.CrawlJob(job_config, url_list)
 
+    # Run display
+    xvfb_wrapper = XvfbWrapper()
+    xvfb_wrapper._from(args.virtual_display)
+    xvfb_wrapper.start()
+
```

Comment:
I tried, but I didn't succeed. Since the command line argument is a string, you should either parse it in the caller (pytbcrawler) and feed it to the XvfbWrapper constructor; or let the constructor handle just the string (since there is no method overloading in python).

I prefer to let the class parse the string and be robust to wrong user input (for instance with some filtering in the _from function) defaulting to the correct values.

Do you see a better option?